test/test_branch.py

```diff
@@ -38,3 +38,15 @@ def test_branch_nogit(git2cpp_path, tmp_path):
     cmd = [git2cpp_path, 'branch']
     p = subprocess.run(cmd, capture_output=True, cwd=tmp_path, text=True)
     assert p.returncode != 0
+
+def tets_branch_new_repo(git2cpp_path, tmp_path, run_in_tmp_path):
```

Comment:
```suggestion
def test_branch_new_repo(git2cpp_path, tmp_path, run_in_tmp_path):
```
Typo in function name.